Automattic/wp-calypso
# Marketplace: Add upsell nudge at plugin categories and search results

#### Proposed Changes

* Added the upgrade nudge at the plugin categories page and the search results page.

#### Testing Instructions

With a simple site without a business plan, test the upsell nudge at:

- /plugins page at `Discover` category
- /plugins page at **any** other category
- /plugins page at the search results page

You should always see this upsell nudge:

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/1044309/194163025-13716cdc-e2aa-4ae7-b2ea-ff158403c408.png">


#### Pre-merge Checklist






Related to 14613-gh-Automattic/woocommerce.com
